### PR TITLE
Style app

### PR DESCRIPTION
**Pull request by paulacarneiro**
Why:
- Design was very simple

How: 
- Use tailwind to style components
- Use svg icons
- Create button to toggle between light and dark mode

### CODE REVIEW

File: components/theme-button.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+  const toggleTheme = () => {
+    setTheme(theme === Theme.Dark ? Theme.Light : Theme.Dark);
+  };
+
+  return (
+    <button
+      onClick={toggleTheme}
+      className="h-8 w-8 overflow-hidden rounded-md bg-light-1 drop-shadow-connect-light  dark:bg-dark-1 dark:shadow-none"
+    >
```
Comment (zediogoviana) [inline components/theme-button.tsx:18]: The preview build is failing because `Missing an explicit type attribute for button`

Comment (paulacarneiro) [reply components/theme-button.tsx]: Fixed it

**Review by zediogoviana — APPROVED** [review]
The code looks good!

Just leaving a couple of suggestions fo the future:

 - Add a label on hover of what each button does
Screenshot 2022-08-30 at 10 40 01

 - Add Placeholders to Search Inputs (or an icon, something like that)
Screenshot 2022-08-30 at 10 40 07

**Review by zediogoviana — APPROVED** [review]
🦭 of approval

**Review by davelange — APPROVED** [review]

**Pull request merged**